custom_components/tuya_local/devices/friedrich_uct08b10a_airconditioner

```diff
@@ -0,0 +1,520 @@
+    # NB: Tuya calls this "Friedrich Window Unit" but these are through-the-wall units
+    name: Friedrich Uni-Fit Through-the-Wall Air Conditioner (Model UCT08B10A)
+
+# Even though the latest firmware for UCT08B10A exposes tons of DPs (very similar to Fisher Summer Air Conditioner)
+# after testing vast majority of them are unsupported. This unit is much closer to the other Friedrich Air Conditioners
+# in terms of actual capabilities but it's still different enough to warrant a separate yaml.
+# Note: we are forced to add all the DPs in order for the YAML configuration to be identified even though most of them
+# need to be made invisible and are not used by the hardware of the actual air conditioner.
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: cold
+              value: cool
+            - dps_val: wet
+              value: dry
+            - dps_val: wind
+              value: fan_only
+            - dps_val: auto
+              value: auto
+            - dps_val: hot
+              value: heat
+              hidden: true
+    - id: 3
+      type: integer
+      name: current_temperature_in_c
+      optional: true
+      unit: C
+    - id: 23
+      name: current_temperature
+      type: integer
+      optional: true
+      hidden: true
+      range:
+        min: -20
+        max: 100
+    - id: 24
+      type: integer
+      name: temperature
+      optional: true
+      unit: F
+      range:
+        min: 690
+        max: 890
+      mapping:
+        - scale: 10
+          step: 10
+    - id: 2
+      name: temperature_in_c
+      type: integer
+      optional: true
+      range:
+        min: 205
+        max: 317
+      mapping:
+        - scale: 10
+          step: 5
+      unit: C
+    - id: 4
+      type: string
+      name: mode
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        # the unit only truly supports 4 speeds: low, med, high, auto
+        - dps_val: auto
+          value: auto
+        - dps_val: low
+          value: low
+        - dps_val: mid
+          value: medium
+        - dps_val: high
+          value: high
+        # the below are all duplicates of some of the above and don't change air speed
+        - dps_val: mute
+          value: quiet
+          optional: true
+          hidden: true
+        - dps_val: mid_low
+          value: medlow
+          optional: true
+          hidden: true
+        - dps_val: mid_high
+          value: medhigh
+          optional: true
+          hidden: true
+        - dps_val: strong
+          value: strong
+          optional: true
+          hidden: true
+    - id: 123
+      name: preset_mode
+      # Tuya IOT dev says this is "bool" but the values aren't boolean
```

Comment:
It is possibly hex encoded along with other values, in which case values of 0 and 1 and a mask of 0x0001 might be better to shield it from changes in the other bits.